packages/hash/frontend/src/blocks/page/MentionView/MentionDisplay.tsx

```diff
@@ -21,7 +21,7 @@ export const MentionDisplay: FunctionComponent<MentionDisplayProps> = ({
   mentionType,
   accountId,
 }) => {
-  const { users, loading: usersLoading } = useUsers();
+  const { users, loading: usersLoading } = useUsers(true);
```

Comment:
I think this is worth doing for now, but I do wonder about the implications of doing this generally. Especially if mentions are expanded to include more than users and pages, if I just made an entity in my page as a result of an action, I'd want it to appear in the mentions (or whatever analogue there is) selector.

I think the longer term solution is making these queries fast enough that it doesn't matter, or to invalidate the cache relatively frequently maybe.

Reply:
Agreed – ideally this would all go through collab